observability/micrometer/src/main/java/org/wildfly/extension/micrometer/registry/WildFlyCompositeRegistry.java

```diff
@@ -0,0 +1,95 @@
+package org.wildfly.extension.micrometer.registry;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.OptionalDouble;
+import java.util.stream.Collectors;
+
+import io.micrometer.core.instrument.FunctionCounter;
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.Tags;
+import io.micrometer.core.instrument.composite.CompositeMeterRegistry;
+import org.jboss.as.controller.client.helpers.MeasurementUnit;
+import org.wildfly.extension.micrometer.MicrometerExtensionLogger;
+import org.wildfly.extension.micrometer.metrics.MetricMetadata;
+import org.wildfly.extension.micrometer.metrics.WildFlyMetric;
+
+public class WildFlyCompositeRegistry extends CompositeMeterRegistry implements WildFlyRegistry {
+    private final List<WildFlyRegistry> registries = new ArrayList<>();
+
+    public WildFlyCompositeRegistry() {
+        super();
+    }
+
+//    @Override
+//    public Meter.Id addMeter(WildFlyMetric metric, MetricMetadata metadata) {
+//        switch (metadata.getType()) {
+//            case GAUGE:
+//                return addGauge(metric, metadata);
+//            case COUNTER:
+//                return addCounter(metric, metadata);
+//            default:
+//                throw MicrometerExtensionLogger.MICROMETER_LOGGER.unsupportedMetricType(metadata.getType().name());
+//        }
+//    }
+
+    private Meter.Id addCounter(WildFlyMetric metric, MetricMetadata metadata) {
```

Comment:
This is unused.

Reply:
Most of that class is no longer used. The class itself may be superfluous. I'll verify then do the needful.